src/Common/src/TypeSystem/Interop/IL/NativeStructType.cs

```diff
@@ -127,7 +127,23 @@ public override TypeSystemContext Context
 
         private NativeStructField[] _fields;
         private InteropStateManager _interopStateManager;
+        private bool _hasInvalidLayout;
 
```

Comment:
Good point. However, `INonEmittableType `is in ILCompiler where as `NativeStructType `is TypeSystem. I'll need to move INonEmittableType to `TypeSystem(src\Common\src\TypeSystem\Common\`).